decryptor/mysql/response_proxy.go

```diff
@@ -101,41 +101,22 @@ const (
 
 // MySQL types
 const (
-	MYSQL_TYPE_NEWDECIMAL byte = iota + 0xf6
```

Comment:
we don't use these constants?

Reply:
yes. maybe I copied them earlier for the future)) but now we don't need them and too lazy was to rename all of them to go style)

Reply:
ok :))